patches/api/0324-Add-more-Campfire-API.patch

```diff
@@ -0,0 +1,58 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: LemonCaramel <admin@caramel.moe>
+Date: Fri, 16 Jul 2021 00:38:52 +0900
+Subject: [PATCH] Add more Campfire API
+
+
+diff --git a/src/main/java/org/bukkit/block/Campfire.java b/src/main/java/org/bukkit/block/Campfire.java
+index f8a344cabb7b5a6d1c5409798a0a98b023bcd756..88d39f0cd819238b24fde042c03f13a5f4124aef 100644
+--- a/src/main/java/org/bukkit/block/Campfire.java
++++ b/src/main/java/org/bukkit/block/Campfire.java
+@@ -69,4 +69,47 @@ public interface Campfire extends TileState {
+      * @param cookTimeTotal Cook time total
+      */
+     void setCookTimeTotal(int index, int cookTimeTotal);
++
++    // Paper start
++    /**
++     * Stop cooking all slot indexes
++     */
++    void disableCooking();
++
++    /**
++     * Resume cooking all slot indexes.
++     */
++    void enableCooking();
++
++    /**
++     * Stop cooking all slot indexes.
++     *
++     * @param index item slot index
++     * @return previous value <br/> <code>true</code> if slot index disable cooking
```

Comment:
the naming of the docs of all return values seems weird, "true if slot index disable cooking" doesn't sound like a proper sentence to me, more like "true if this slot index disables cooking" of "true if cooking is disabled for this slot", (this would sound nicer if the stop naming I proposed, "true if cooking is stopped for this slot inded")